XenAdmin/Commands/ConvertToThinSRCommand.cs

```diff
@@ -98,7 +98,7 @@ private SR GetFirstThickSRFromSelection(SelectedItemCollection selection)
             {
                 var sr = item.XenObject as SR;
 
-                if (sr != null && Helpers.DundeeOrGreater(sr.Connection) && sr.Provisioning == SrProvisioning.Thick && (sr.type == "lvmohba" || sr.type == "lvmoiscsi"))
+                if (sr != null && Helpers.DundeeOrGreater(sr.Connection) && (sr.type == "lvmohba" || sr.type == "lvmoiscsi") && !sr.IsThinProvisioned)
```

Comment:
I suggest removing the SR.Provisioning method, which is no longer used, to avoid confusion in future.